src/main/java/uk/nhs/tis/trainee/notifications/dto/ProgrammeMembershipEvent.java

```diff
@@ -21,19 +21,17 @@
 
 package uk.nhs.tis.trainee.notifications.dto;
 
-import java.time.LocalDate;
-import java.util.List;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
 
 /**
  * A Programme Membership event.
  *
- * @param tisId               The TIS ID of the Programme membership.
- * @param startDate           The start date of the Programme membership.
- * @param curricula           The list of curricula for the Programme membership.
+ * @param tisId The TIS ID of the Programme membership.
+ * @param recrd The record DTO map.
  */
 public record ProgrammeMembershipEvent(
     String tisId,
-    LocalDate startDate,
-    List<Curriculum> curricula) {
+    @JsonProperty("record") RecordDto recrd) implements Serializable {
```

Comment:
I was hoping to avoid dealing with the generic record to help move away from it down the road, needs must though.